pkg/descheduler/evictions/evictions.go

```diff
@@ -74,6 +75,7 @@ func NewPodEvictor(
 	evictSystemCriticalPods bool,
 	ignorePvcPods bool,
 	evictFailedBarePods bool,
+	metricsEnabled bool,
```

Comment:
why this?

Reply:
To avoid accumulating the metrics in case the metrics server is not enabled. To save some instructions.